public/cantusdata/test/management/commands/test_import_all_data.py

```diff
@@ -9,7 +9,8 @@ def setUp(self):
         pass
 
     def test_import_all_data(self):
-        call_command('import_data', 'test/test_chants.csv', '1', all=True)
+        call_command('import_data', 'concordances')
+        call_command('import_data', 'manuscripts')
```

Comment:
The original code seems to be importing chants, whereas the new one `concordances` and `manuscripts`. Is that the expected behaviour? That is, will the unit tests pass/fail accordingly?

Reply:
The other tests are not using the data that is imported here. The point of this specific test is to test the ability to import data and as soon as it's done it deletes everything. I changed it from chants to concordances and manuscripts as you can't import chants independently anymore, nor does it seem like you can provide a directory as an argument to import stuff from.

Reply:
Yes, you can't import chants independently anymore. If the purpose is importing data and deleting it right after, then this is perfect. Thanks!